stdlib/urllib/parse.pyi

```diff
@@ -132,14 +133,14 @@ def parse_qsl(
     separator: str = ...,
 ) -> list[tuple[AnyStr, AnyStr]]: ...
 @overload
-def quote(string: str, safe: str | bytes = ..., encoding: str | None = ..., errors: str | None = ...) -> str: ...
+def quote(string: str, safe: str | ReadableBuffer = ..., encoding: str | None = ..., errors: str | None = ...) -> str: ...
```

Comment:
From the link you posted, `safe` is `str | Iterable[int]` instead. The code doesn't use the buffer protocol. For example, `urllib.parse.quote("x", safe=pickle.PickleBuffer(b"x"))` gives `TypeError: 'pickle.PickleBuffer' object is not iterable`.